ERCS/erc-iga.md

```diff
@@ -0,0 +1,170 @@
+---
+eip: <to be assigned>
```

Comment:
```suggestion
eip: 8097
```
Assigning next sequential EIP/ERC/RIP number.
Numbers are assigned by editors & associates.

Please also update the filename.